src/object/srv_obj_migrate.c

```diff
@@ -42,6 +42,7 @@
 
 /* This needs to be here to avoid pulling in all of srv_internal.h */
 int ds_cont_tgt_destroy(uuid_t pool_uuid, uuid_t cont_uuid);
+int ds_cont_tgt_force_close(uuid_t cont_uuid);
```

Comment:
(style)  externs should be avoided in .c files

Reply:
Needed because container srv_internal.h is not included here.